detekt.properties

```diff
@@ -1,12 +1,18 @@
 category=External Analysers
 description=Provide detekt rules for Kotlin projects
 homepageUrl=https://github.com/detekt/sonar-kotlin
-archivedVersions=
-publicVersions=2.0.0
+archivedVersions=2.0.0
+publicVersions=2.1.0
 
 defaults.mavenGroupId=io.github.detekt
 defaults.mavenArtifactId=sonar-detekt
 
+2.1.0.description=Upgrade to detekt 1.9.1
+2.1.0.sqVersions=[7.9,LATEST]
+2.1.0.date=2020-06-02
+2.1.0.changelogUrl=https://github.com/detekt/sonar-kotlin/releases/tag/2.1.0
+2.1.0.downloadUrl=https://github.com/detekt/sonar-kotlin/releases/download/2.1.0/sonar-detekt-2.1.0.jar
+
 2.0.0.description=Initial Marketplace entry
 2.0.0.sqVersions=[7.9,LATEST]
```

Comment:
Only one version can be compatible with `LATEST`. You should set this to 8.3.

Reply:
done.